scss/components/_list-styles.scss

```diff
@@ -78,6 +78,23 @@
   }
 }
 
+.list--flat-bordered {
+  @extend .list--flat;
+  padding-bottom: u(.5rem);
+
+  li {
+    &::after {
```

Comment:
I can't show a screenshot right now since my fec-cms/style local is busted, but because of the way the padding/margins are set, borders would be longer than the text height.